src/vector/jitsi/index.ts

```diff
@@ -75,6 +77,12 @@ let widgetApi: WidgetApi;
             await widgetApi.setAlwaysOnScreen(false); // start off as detachable from the screen
         }
 
+        widgetApi.addTerminateCallback(() => {
+            if (terminateMeeting) {
+                return terminateMeeting();
```

Comment:
Would be good to have some logging just above this to say that we're intentionally hanging up the call.

Reply:
Ok, added logging here.